README.adoc

```diff
@@ -25,38 +26,75 @@ endif::[]
 
 = UnifiedPush Operator
 
-The UnifiedPush Operator for Kubernetes provides an easy way to
-install and manage an AeroGear UnifiedPush Server on Kubernetes.
-
 toc::[]
 
-== Limitations
+== Overview
 
-// https://issues.jboss.org/browse/AEROGEAR-9162
-[NOTE]
-====
-This operator currently only works on OpenShift. This is because it
-provisions `Route`s and `ImageStream`s, and also relies on the fact that OpenShift
-provides an OAuth server. In future we aim to make it work on vanilla
-Kubernetes also.
-====
+The UnifiedPush Operator for Kubernetes provides an easy way to install and manage an https://aerogear.org/docs/unifiedpush/[AeroGear UnifiedPush] Server on OpenShift.
+
+== Prerequisites
 
-== Installation
+|===
+|https://golang.org/doc/install[Install Golang]
+|https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
+|https://golang.github.io/dep/docs/installation.html[Install the dep package manager]
+|https://github.com/operator-framework/operator-sdk#quick-start[Install Operator-SDK]
+|https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl[Install kubectl]
+|===
 
-As a user with admin permissions, you can easily install the
-unifiedpush-operator in your OpenShift cluster as follows:
+== Getting Started
 
-....
-kubectl create namespace unifiedpush
-kubectl create -n unifiedpush -f deploy/service_account.yaml
-kubectl create -n unifiedpush -f deploy/role.yaml
-kubectl create -n unifiedpush -f deploy/role_binding.yaml
-kubectl create -n unifiedpush -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
-kubectl create -n unifiedpush -f deploy/crds/push_v1alpha1_pushapplication_crd.yaml
-kubectl create -n unifiedpush -f deploy/crds/push_v1alpha1_androidvariant_crd.yaml
-kubectl create -n unifiedpush -f deploy/crds/push_v1alpha1_iosvariant_crd.yaml
-kubectl create -n unifiedpush -f deploy/operator.yaml
-....
+=== Cloning the repository
+
+By the following commands you will create a local directory and clone this project.
+
+[source,shell]
+----
+$ git clone git@github.com:aerogear/unifiedpush-operator.git $GOPATH/src/github.com/aerogear/unifiedpush-operator
+----
+
+=== Minishift installation and setup
+
+https://docs.okd.io/latest/minishift/getting-started/installing.html[Install Minishift] then enable Operators on it by running the following commands.
+
+[source,shell]
+----
+# create a new profile to test the operator
+$ minishift profile set unifiedpush-operator
+
+# enable the admin-user add-on
+$ minishift addon enable admin-user
+
+# add insecure registry to download the images from docker
+$ minishift config set insecure-registry 172.30.0.0/16
+
+# start the instance
+$ minishift start
+----
+
+IMPORTANT: In OpenShift 4 the operators were introduced by default.
+
+=== Installing
+
+Use the following command to install the unifiedpush-operator in your OpenShift cluster as follows:
```

Comment:
I saw other places in the readme use `UnifiedPush Operator`, so it might be better to use one or the other to keep the name consistent?

Reply:
👍